pkg/blockstorage/azure/client.go

```diff
@@ -64,95 +74,78 @@ func NewClient(ctx context.Context, config map[string]string) (*Client, error) {
 			return nil, errors.Wrap(err, "Cannot get subscriptionID from instance metadata")
 		}
 	}
+	/*
```

Comment:
shouldn't we remove these commented code.

Reply:
Done